lib/app/utils/terminology.rb

```diff
@@ -233,11 +233,16 @@ def self.missing_validators
     # @param String code the code to validate against the valueset
     # @param String system an optional codesystem to validate against. Defaults to nil
     # @return Boolean whether the code or code/system is in the valueset
-    def self.validate_code(valueset_url, code, system = nil)
+    def self.validate_code(valueset_url: nil, code:, system: nil)
       # Get the valueset from the url. Redundant if the 'system' is not nil,
       # but allows us to throw a better error if the valueset isn't known by Inferno
-      validation_fn = FHIR::StructureDefinition.vs_validators[valueset_url]
-      raise(UnknownValueSetException, valueset_url) unless validation_fn
+      if valueset_url
+        validation_fn = FHIR::StructureDefinition.vs_validators[valueset_url]
+        raise(UnknownValueSetException, valueset_url) unless validation_fn
+      else
+        validation_fn = FHIR::StructureDefinition.vs_validators[system]
+        raise(Inferno::Terminology::ValueSet::UnknownCodeSystemException, system) unless validation_fn
```

Comment:
![Screen Shot 2020-04-02 at 2 40 07 PM](https://user-images.githubusercontent.com/15969967/78289388-216d3680-74f0-11ea-8167-0a0959aed59e.png)
Looks like it's `Valueset` rather than `ValueSet`. There's also a mis-cased reference in line 209.

Reply:
I swear to god I fat finger this every time. I'm going to make a PR to rename this class, because it doesn't follow the convention of every other class in our repo.

Reply:
Yes, please. `Valueset` is very confusing.

Reply:
Fixed; next reviewer should verify (do a case-sensitive search for `Valueset`, and ensure there aren't any references to the class`.